src/common/SurgeSynthesizerIO.cpp

```diff
@@ -200,6 +201,20 @@ void SurgeSynthesizer::loadRaw(const void* data, int size, bool preset)
    halt_engine = false;
    patch_loaded = true;
    refresh_editor = true;
+	if (patchid < 0)	//category.size() == 0)
+	{//JS: it is a setchunk so I look up the set current category and patch
```

Comment:
I don't understand this sentence.

Reply:
I changed the comment in the latest commit but basically setchunk is the method used by the host to send the plug-ins data to the plug-in to completely restore the saved patch.